exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/ExtendedOptionIterator.java

```diff
@@ -52,14 +55,44 @@
  *  only the value set at SESSION level.
  */
 public class ExtendedOptionIterator implements Iterator<Object> {
-//  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(OptionIterator.class);
+  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ExtendedOptionIterator.class);
+  private static final String OPTIONS_DESCRIBE_PROPERTIES = "options.describe.properties";
+  private static final Properties descriptionMap = new Properties();
 
   private final OptionManager fragmentOptions;
   private final Iterator<OptionValue> mergedOptions;
+  private HashMap<OptionValue.Kind,String> typeMapping;
```

Comment:
Can we use more generic `Map`, plus space after comma.

Reply:
Ok, just moved the method out from where it originally was. Will make the change